lib/YaST/Bootloader/BootLoaderSettingsPage.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces all accessing methods for Bootloader settings page
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Bootloader::BootLoaderSettingsPage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my ($self) = @_;
+    $self->SUPER::init();
+    $self->{tb_boot_loader_settings} = $self->{app}->tab({id => '"CWM::DumbTabPager"'});
```

Comment:
it would be more simple to include this control in each page (not even worth it to inherit IMO) because I think it is tricky to know where you are in which tab or which tab is selected by default when you land (referring to the `is_shown` method for example), this is part of the flow of the controller for each product to use a series of steps needed.